.eslintrc

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Why do you have this file in your PR?

I think you this PR contains some of the files in the previous PR.

Reply:
I started working on this branch before the eslint PR was merged and so I had to pull origin staging into this branch and the eslint from staging appeared s a new file.

Reply:
I will fix it now.